src/app/admin-patients-list/admin-patients-list.component.ts

```diff
@@ -0,0 +1,78 @@
+import { Component, OnInit, Input, Inject } from '@angular/core';
+import { PatientResponse } from '../../../api-objects/PatientResponse';
+import { CustomerSearchService } from '../customer-search/customer-search.service';
+import { MatDialog, MAT_DIALOG_DATA } from '@angular/material';
+import { AdminPatientListModalComponent } from './admin-patient-list-modal/admin-patient-list-modal.component';
+
+@Component({
+  selector: 'admin-patients-list',
+  templateUrl: './admin-patients-list.component.html',
+  styleUrls: ['./admin-patients-list.component.css']
+})
+export class AdminPatientsListComponent implements OnInit {
+  public patients: Array<PatientResponse> = [];
+  public currentPage: number = 0;
+
+  constructor(
+    public dialog: MatDialog
+  ) { }
+
+  public ngOnInit() {
+    this.loadPatients();
```

Comment:
I think `loadPatients` should be called when the user clicks the `Patients` button in navigation.

Reply:
Okay, patients are now loaded when navigation tab is clicked.